test/powershell/Language/Parser/Parsing.Tests.ps1

```diff
@@ -302,3 +302,11 @@ Describe 'expressions parsing' -Tags "CI" {
 Describe 'Hash Expression parsing' -Tags "CI" {
     ShouldBeParseError '@{ a=1;b=2;c=3;' MissingEndCurlyBrace 2
 }
+
+Describe 'Unicode escape sequence parsing' -Tag "CI" {
+    ShouldBeParseError '"`u{110000}"' InvalidUnicodeEscapeSequenceValue 4 # error offset is "`u{>>1<<10000}"
+    ShouldBeParseError '"`u{1234567}"' UnicodeEscapeSequenceMissingTerminator 10 # error offset is "`u{123456>>7<<}"
+    ShouldBeParseError '"`u219z"' InvalidUnicodeEscapeSequence 1
+    ShouldBeParseError '"`u{219z"' UnicodeEscapeSequenceMissingTerminator 7 # error offset "`u{219>>z<<"
+    ShouldBeParseError '"`u{}"' InvalidBracketedUnicodeEscapeSequence 1
+}
```

Comment:
Yeah, I've noticed that handy little feature of PSReadline.  Will add these additional tests.  Do you have an opinion on whether I should test single quoted strings/here strings?  In theory, none of this new code should be invoked for sq strings.